app/presenters/api/v2/headings/declarable_heading_presenter.rb

```diff
@@ -13,6 +17,8 @@ def initialize(heading, measures)
           @export_measures = measures.select(&:export).map do |measure|
             Api::V2::Measures::MeasurePresenter.new(measure, heading)
           end
+          @unit_measures = @import_measures.select(&:expresses_unit?)
+          @third_country_measures = import_measures.select(&:third_country?)
```

Comment:
`@import_measures` in line 20
and 
`import_measures` in line 21
to be consistent let's chose one
maybe `import_measures` ?

Reply:
```suggestion
          @unit_measures = import_measures.select(&:expresses_unit?)
          @third_country_measures = import_measures.select(&:third_country?)
```